posts/inside-rust/2021-01-19-changes-to-rustdoc-team.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Changes to Rust rustdoc team"
+author: Guillaume Gomez
+description: "leadership and membership additions"
+team: the rustdoc team <https://www.rust-lang.org/governance/teams/dev-tools#rustdoc>
+---
+
+Recently, there have been a lot of improvements in rustdoc. It was possible thanks to our new contributors. In light of these recent contributions, a few changes were made in the rustdoc team.
+
+## Adding a new team leader
+
+[Joshua](https://github.com/jyn514) has been helping me **a lot** on the leading parts by tagging issues and mentoring newcomers. But at the same time, he initiated great changes in the rustdoc codebase that led to huge performance improvements, he finished the "intra doc link" features which is now stable after years of instability. He also did multiple additional things but let's stop the listing here. :)
+
+Overall, he's been a monstruous help and making him co-lead is just the officialization of everything's he's been doing since he joined the rustdoc team (and even before!) and I'm glad to have him on the team and as co-lead.
```

Comment:
Heh, I hope my help is not "extremely and dauntingly large" :laughing:
```suggestion
Overall, he's been a enormous help and making him co-lead is just making offical everything's he's been doing since he joined the rustdoc team (and even before!). I'm glad to have him on the team and as co-lead.
```